apistructs/cluster_info.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/erda-project/erda/pkg/strutil"
 )
 
+const (
+	ClusterInfoConfigMapName = "dice-cluster-info" //ClusterInfoConfigMapName cluster info configmap name
```

Comment:
ConfigMapNameOfClusterInfo
ConfigMapNameOfAddons

and add space between `//Clus...`

Reply:
adjusted